### PR TITLE
new cards for CPhPlus2jets for 2016

### PR DESCRIPTION
**Pull request by gourangakole**
Cards are for 6 signal samples, 3 inclusive gg->h production and 3 for gg->h+2jet production. For each production there are 3 options, scalar Higgs, Pseudoscalar Higgs and Mixing CP Higgs.  Samples to be generated with Higgs Characterization package of MG5_aMC@NLO and for FxFx matching

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/Higgs/GluGluToHToTauTauPlusTwoJets_M125_13TeV_amcatnloFXFX_pythia8/GluGluToHToTauTauPlusTwoJets_M125_13TeV_amcatnloFXFX_pythia8_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+#*********************************************************************** 
+# Reweight variables for scale dependence and PDF uncertainty          *
+#***********************************************************************
+ 1.0, 2.0, 0.5 = rw_rscale ! muR factors to be included by reweighting
+ 1.0, 2.0, 0.5 = rw_fscale ! muF factors to be included by reweighting
+ True = reweight_scale ! Reweight to get scale variation using the 
+            ! rw_rscale and rw_fscale factors. Should be a list of 
+            ! booleans of equal length to dynamical_scale_choice to
+            ! specify for which choice to include scale dependence.
+ #$DEFAULT_PDF_MEMBERS = reweight_PDF     ! reweight to get PDF uncertainty
```
Comment (efeyazgan) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/Higgs/GluGluToHToTauTauPlusTwoJets_M125_13TeV_amcatnloFXFX_pythia8/GluGluToHToTauTauPlusTwoJets_M125_13TeV_amcatnloFXFX_pythia8_run_card.dat]: please remove this unused line

**Comment by efeyazgan** [comment]
Do you need the following lines?
define l+ = e+ mu+
define l- = e- mu-
define vl = ve vm vt
define vl~ = ve~ vm~ vt~

**Comment by anikiten** [comment]
these lines are always there generated by MG5 by default. They are not needed for this production and it does not matter to keep them or remove

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/Higgs/GluGluToHToTauTauPlusTwoJets_M125_13TeV_amcatnloFXFX_pythia8/GluGluToHToTauTauPlusTwoJets_M125_13TeV_amcatnloFXFX_pythia8_run_card.dat (hunk 1 above)
Comment (anikiten) [reply bin/MadGraph5_aMCatNLO/cards/production/13TeV/Higgs/GluGluToHToTauTauPlusTwoJets_M125_13TeV_amcatnloFXFX_pythia8/GluGluToHToTauTauPlusTwoJets_M125_13TeV_amcatnloFXFX_pythia8_run_card.dat]: does it mean that 6 gridpacks must be regenerated again w/o this line ? I will take about of 5 days...

**Comment by efeyazgan** [comment]
No, that's OK. (We usually remove unused definitions. )

**Pull request merged**